tests/departements/eure_et_loir/pch_etablissement.yml

```
@@ -0,0 +1,153 @@
+        nationalite: LL
+        titre_sejour: True
+        handicap: True
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: True
+
+- name: Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH refugie
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: LY
+        titre_sejour: False
+        refugie: True
+        handicap: True
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: True
+
+- name: Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH apatride
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: LY
+        titre_sejour: False
+        apatride: True
+        handicap: True
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: True
+
+- name: Non Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH nationalité non UE et titre séjour KO
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: LL
+        titre_sejour: False
+        handicap: True
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: False
+
+- name: Non Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH non UE titre séjour KO non refugie non apatride
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: LL
+        titre_sejour: False
+        refugie: False
+        apatride: False
+        handicap: True
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: False
+
+- name: Non Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH non handicapée
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: FR
+        titre_sejour: False
+        handicap: False
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: False
+
+- name: Non Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH sans place hébergement
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: FR
+        titre_sejour: False
+        handicap: True
+    famille:
+      place_hebergement: False
+    menage:
+      depcom: 28000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: False
+
+- name: Non Éligibilité à l'aide PCH de l'Eure-et-Loir en établissement avec PH hors 28
+  period: 2020-10
+  input:
+    individus:
+      p1:
+        nationalite: FR
+        titre_sejour: False
+        handicap: True
+    famille:
+      place_hebergement: True
+    menage:
+      depcom: 45000
+  output:
+    eure_et_loir_eligibilite_pch_etablissement: False
```

Ajout d'un retour à la ligne 🙂 
 
```suggestion
    eure_et_loir_eligibilite_pch_etablissement: False

```